lib/ProductOpener/Food.pm

```diff
@@ -1195,38 +1178,34 @@ sub compute_nutriscore_2021_fruits_vegetables_nuts_colza_walnut_olive_oil ($prod
 		return $nutriscore_category_override_for_fruits_vegetables_legumes;
 	}
 
-	if (defined $product_ref->{nutriments}{"fruits-vegetables-nuts-dried" . $prepared . "_100g"}) {
-		$fruits = 2 * $product_ref->{nutriments}{"fruits-vegetables-nuts-dried" . $prepared . "_100g"};
+	my $fruits_vegetable_nuts
+		= deep_get($product_ref, "nutrition", "aggregated_set", "nutrients", "fruits-vegetables-nuts", "value");
```

Comment:
we loose the "prepared" notion because for a product that needs to be prepared, we put "prepared" values in aggregated_set ?

Reply:
It would be worth reminding this in a comment.